extensions/amp-a4a/0.1/real-time-config-manager.js

```diff
@@ -63,16 +68,45 @@ export const RTC_ERROR_ENUM = {
 /**
  * @param {string} error
  * @param {string} callout
+ * @param {!Window} win
+ * @param {!../../../src/service/ampdoc-impl.AmpDoc} ampDoc
  * @param {number=} opt_rtcTime
  * @return {!Promise<!rtcResponseDef>}
  * @private
  */
-function buildErrorResponse_(error, callout, opt_rtcTime) {
+function buildErrorResponse_(
+  error, callout, errorReportingUrl, win, ampDoc, opt_rtcTime) {
   dev().warn(TAG, `RTC callout to ${callout} caused ${error}`);
+  if (errorReportingUrl) {
+    sendErrorMessage(error, errorReportingUrl, win, ampDoc);
+  }
   return Promise.resolve(/**@type {rtcResponseDef} */(
     {error, callout, rtcTime: opt_rtcTime || 0}));
 }
 
+/**
+ * @param {string} errorType Uses the RTC_ERROR_ENUM above.
+ * @param {string} errorReportingUrl
+ * @param {!Window} win
+ * @param {!../../../src/service/ampdoc-impl.AmpDoc} ampDoc
+ */
+export function sendErrorMessage(errorType, errorReportingUrl, win, ampDoc) {
+  if (ERROR_REPORTING_ENABLED || getMode(win).localDev || getMode(win).test) {
+    if (!isSecureUrl(errorReportingUrl)) {
+      dev().warn(TAG, `Insecure RTC errorReportingUrl: ${errorReportingUrl}`);
+      return;
+    }
+    const whitelist = {ERROR_TYPE: true, HREF: true};
+    const macros = {
+      ERROR_TYPE: errorType,
+      HREF: win.location.href,
+    };
+    const url = Services.urlReplacementsForDoc(ampDoc).expandUrlSync(
+        errorReportingUrl, macros, whitelist);
+    sendRequest(win, url, {image: true});
```

Comment:
If I understand this correct, this will always send a request with image. Then why importing from `transport.js`, we can simply create `new Image()` with src equals to url. Let me know if that works. Thanks.

Reply:
done